client/blocks/plan-purchase-features/features-list.jsx

```diff
@@ -0,0 +1,275 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+import GoogleVoucherDetails from 'my-sites/upgrades/checkout-thank-you/google-voucher';
+import QuerySiteVouchers from 'components/data/query-site-vouchers';
+import HappinessSupport from 'components/happiness-support';
+import PlanIcon from 'components/plans/plan-icon';
+
+export const FindNewThemeFeature = localize( ( { selectedSite, translate } ) => {
+	return (
+		<div className="plan-purchase-features__item">
+			<PurchaseDetail
+				icon="customize"
+				title={ translate( 'Find a new theme' ) }
+				description={ translate( 'All our premium themes are now available at no extra cost. Try them out now.' ) }
+				buttonText={ translate( 'Browse premium themes' ) }
+				href={ '/design/' + selectedSite.slug }
+			/>
+		</div>
+	);
+} );
+
+export const AdvertisingRemovedFeature = localize( ( { isBusinessPlan, translate } ) => {
```

Comment:
Let's split these out into their own files.  Are you running into any eslint issues with the `plan-purchase-features__item` class?